src/taco-livereload/lib/browserSyncServer.js

```diff
@@ -0,0 +1,134 @@
+/**
+ *******************************************************
+ *                                                     *
+ *   Copyright (C) Microsoft. All rights reserved.     *
+ *                                                     *
+ *******************************************************
+ */
+
+
+var path = require('path');
+var Q = require('q');
+var fs = require('fs');
+var url = require('url');
+var logger = require('./utils/logger');
+var helpers = require('./utils/helpers');
+var merge = require('lodash.merge');
+
+var projectRoot;
+
+/**
+ * @constructor
+ * @param {string} projectRoot - root folder of the project
+ * @param {Object} options - Options to initialize LiveReload with - These are to be passed to the underlying BrowserSync.
+ *                           See http://www.browsersync.io/docs/options/ for documentation
+ */
+
+function BrowserSyncServer(projectRoot, options) {
+    this.projectRoot = projectRoot;
+    this.options = options;
+    this.browserSync = require('browser-sync').create(); 
```

Comment:
It's hard to find this dependency among the others. Might it be better to place it at the top with the others?